pkg/addon/parameters.go

```diff
@@ -43,26 +44,31 @@ func GetParameterByInstallation(ctx context.Context, client k8sclient.Client, in
 	)
 }
 
-// GetParameterByInstallType retrieves the value for an addon parameter by
-// selecting the addon name for installationType
+// GetParameterByInstallType retrieves the value for an addon parameter from
+// the addon parameters secret
 func GetParameterByInstallType(ctx context.Context, client k8sclient.Client, installationType integreatlyv1alpha1.InstallationType, namespace, parameter string) ([]byte, bool, error) {
-	addonName := GetName(installationType)
-	secretName := fmt.Sprintf("addon-%s-parameters", addonName)
-
-	secret := &corev1.Secret{}
-	if err := client.Get(ctx, k8sclient.ObjectKey{
-		Name:      secretName,
-		Namespace: namespace,
-	}, secret); err != nil {
+	secrets := &corev1.SecretList{}
+	opts := []k8sclient.ListOption{
+		k8sclient.InNamespace(namespace),
+	}
```

Comment:
We should add some filtering on labels here to reduce the number of secrets we need to loop over.